source/main/modules/window-manager/index.ts

```diff
@@ -864,4 +905,4 @@ export default class WindowManager {
     // 0 = Ok, 1 = Cancel
     return (response.response === 0)
   }
-}
+}
```

Comment:
Fails `yarn lint` : `  908:2   error    Newline required at end of file but not found  eol-last`